src/mcts/old_test.cc

```diff
@@ -0,0 +1,290 @@
+/**
```

Comment:
is this file unused? i'd just delete this file if you're not using it

Reply:
I see, in that case I'd leave a TODO comment at the top of the file saying that this file isn't in use but we should get it working in the future

Reply:
I simply raised an issue